### ISSUE TITLE
Logo Issue & Front Page Format Example 1.3.0

### ISSUE THREAD

**Issue by jpeizer1** [OpenDevelopmentMekong/opendev-wptheme#358]
@kathi-aeviator As raised previously two weeks back the logo was changed in this redesign and Dan and I were informed it was "out of design standard" for our logos. Our current production page design represents the correct logos for Mekong and all countries and the new format should reflect those as closely as possible (@Dan I assume this is the standard) 

![logo](https://cloud.githubusercontent.com/assets/16559612/12216812/48403d8a-b6bc-11e5-8867-eb78ac4a8ca3.jpg)

**Comment by kathi-aeviator** [comment]
From the technical side it's easy to make and from the design-side it needs a bit more Space between the sign and the text. The navigation hides while scrolling and it's important to have the sign on the left side then to let the user know in which country he is!

**Comment by kathi-aeviator** [comment]
SOLUTION:
1. minimize and move  design-mark
.mainNav-logo {
margin: 12px 60px;
float: none;
font-size: 60px;
1. marginally move word-mark

delete margin-left from word-mark

![image](https://cloud.githubusercontent.com/assets/15011163/12218124/0d1f4930-b716-11e5-8630-ffb1fde89d86.png)
1. slim the left country-side bar as much as possible

-> 110 px widh

**Comment by kathi-aeviator** [comment]
@acorbi are you able to manage that or should I ask Chris in the end of mapexplorer-PKG?

**Comment by acorbi** [comment]
@kathi-aeviator Please coordinate with @chris-aeviator.This should be done quite quickly, and as discussed with him, I only will help out with the CKAN part of PK1. 

I suggest that @chris-aeviator does it as a little task to make a break for 5-10 minutes from the implementation of map-explorer. This is the way I do it most of the times. I use little tasks (where one does not need to think that much) to make a break from other more demanding operations....

**Comment by jpeizer1** [comment]
If there is a little extra space between logo and words it's ok. @chris-aeviator can you handle this at end of map explorer work all the country logo screens need to change as well so they are consistent.

**Comment by jpeizer1** [comment]
I had to go out for food so answering on mobile -- if there is some extra space between logo and words it's not such a problem I think

Rgds

JP

**Comment by kathi-aeviator** [comment]
assigned chris for this issue.

**Comment by jpeizer1** [comment]
As per Kathi's suggestion we comment here: In addition we'd like the live site Main pages (Mekong/Cambodia/Vietnam? @DBishton please confirm for Vietnam and what we do with their extra bottom feature widget)  to look like this:

![mockup5](https://cloud.githubusercontent.com/assets/16559612/12323644/2c2fb85a-ba8b-11e5-8487-0cf80383bfb2.jpg)

This mockup basically places the current production home page slider over the new news container as the only major modification so items other than news articles as well can be FEATURED. I have tried to make it as close to the current iteration of the home page page as possible but with the official logo already modified to reflect the standard - The logo should look as close to what the current production pages show as possible for each country. https://opendevelopmentmekong.net

Dan and Pinkie's basically have both indicated they are fine with the proposed layout in the attached mockup image, as long as the full-length news box could be styled to include photos. Dan is good with that too his questions are more around how its administrated -- but thinks what is being suggested in Mockup  looks really good.

PLEASE Note the mockup reflects NEWS CONTAINER  version at the time the image was created and we recognize that may change a bit still - Like Pinkie's request of  PHOTOS included in the full news page news box.

**Comment by kathi-aeviator** [comment]
@jpeizer1 just that we not missunderstanding each other. You are talking just about
 the Logo, did you? If this is right I can give css proposals to the dev team.

**Comment by chris-aeviator** [comment]
@jpeizer1 

can you please expain a bit more on your suggestion with the mockup? Are you talking about implementing news explorer & slideshow on vietnam site (including images) ?

**Comment by chris-aeviator** [comment]
implementing logo..

**Comment by jpeizer1** [comment]
@chris-aeviator @kathi-aeviator  Ok -- Kathi asked me to put the logo and the mockup issue together in this issue - they are related but separate issues. 

LOGOS: All the pages with logos now in the new design should have logos that conform as much as they can to the various country logos on each PRODUCTION page of the current system. So current production page Vietnam logo to NEW PAGE Vietnam logo, Cambodia production page logo to NEW PAGE Cambodia Logo etc. Basically keep the logos the same as they were on the old system because they conformed to the standard.

MOCKUP -- I understand MEKONG and CAMBODIA will go live with these type of page I suggest (with Slider and Huy Engs new News container). VIETNAM is a partner country and if you look at their Production page right now you will see an equivalent type of page - It has a slider on top, and a map/news container under it and one extra widget (events?)  under that. It is my impression then that when we go live with the configuration I suggested for MEKONG and CAMBODIA  that VIETNAM has it too. I would just like @DBishton to confirm and also ask what he feels we should do about their extra widget at the bottom.

**Comment by kathi-aeviator** [comment]
@jpeizer1 I understand, thanks for making that clear.

LOGOS: We implemented standard-Logos to all countrysides. Not deployed yet.

opened issue for CKAN Logo:
https://github.com/OpenDevelopmentMekong/ckanext-odm_nav/issues/38

Closing issue after deploying.

MOCKUPS: made a new branch for the mockup issue here:
https://github.com/OpenDevelopmentMekong/opendev-wptheme/issues/427

sorry JP for missunderstanding!
